website/content/docs/concepts/billing-start-date.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: Billing Start Date
+description: >-
+   A description of Vault cluster's billing start date used throughout Vault.
+---
+
+# Billing Start Date
+
+This is the billing start date associated with the reporting cluster. This will be the license start date if not configured.
```

Comment:
```suggestion
Billing start date reflects the current Vault license `StartTime` field.

Vault uses the billing start date as a default point of reference for license utilization queries, including:
1. Direct queries via the activitylog API (TODO: link to API docs)
2. activity export CLI
3. Automated License Utilization reports (link, if relevant)
4. Manual License Utilization Reports (link)
```

Reply:
The billing start date can also be configured via the config file. We do not show this field in documentation. I am not sure if we need to cover the config case here too.

Reply:
Optimally, we would correct the gap in the config docs and then document the full options in a related how-to guide

Reply:
But I think we don't want to document about the billing start time field in config. I don't know when that decision was made probably when the initial census changes went out.